fp8/flux_pipeline.py

```diff
@@ -386,12 +386,12 @@ def into_bytes(self, x: torch.Tensor, jpeg_quality: int = 99) -> io.BytesIO:
         return im
 
     @torch.inference_mode()
-    def as_img_tensor(self, x: torch.Tensor) -> io.BytesIO:
-        """Converts the image tensor to bytes."""
```

Comment:
extremely nit but might be worth renaming this method to `as_pil_and_np_image` or something.